client/scss/components/forms/_form-width.scss

```diff
@@ -15,6 +15,10 @@
       width: theme('width.[3/5]');
     }
   }
+
+  .w-tabs__panel {
+    margin-inline-end: calc(var(--side-panel-width) - 420px);
+  }
```

Comment:
I would recommend against an additional global style here, this does not account for the following cases.

1. Side panel was open but is now closed (custom property will be set but we no longer wand this margin set)
2. Side panel AND minimap is open - does not account for the dynamic widths (see above)
3. Does not account for there not being enough room for the actual tab content, at some point close to the lower end of the md breakpoint and when the sidebar is super wide, we will want it to simply overlap and that's ok. It's better than the content being super squished.

Here's a different starting point that accounts for all of these,

```scss

@include media-breakpoint-up(md) {
  .minimap-open {
    .tab-content {
      width: theme('width.[4/5]');
    }
  }

  .side-panel-open {
    .tab-content {
      // Account for dynamic width of the side panel when open.
      width: max(theme('width.[1/3]'), calc(100% - var(--side-panel-width)));
    }
  }

  .side-panel-open.minimap-open {
    .tab-content {
      // Account for additional space taken up by the minimap.
      width: max(theme('width.[2/5]'), calc(100% - var(--side-panel-width) - 15rem));
    }
  }
}
```

However, this also has some rough edges.

1. We have to hard code an assumed value for the minimap width, 15 rem is roughly 260px. see `$minimap-width` but I am not sure  if this is what we want to do, it's probably OK for the minimap to overlap somewhat when the side panel is also open.
2. The mix of ratios and concrete values can still cause some odd cases, needs to be validated with a wide range of viewports from md up.

Reply:
Let me give these a try and see how it is.